README.md

```diff
@@ -51,6 +51,37 @@ npm install --save ts-json-schema-generator
     Do not sort properties.
 ```
 
+## Programmatic Usage
+
+Here is an example of using it programmatically:
+
+```typescript
+import { SchemaGenerator } from "ts-json-schema-generator";
+import * as ts from "typescript";
```

Comment:
Does typescript have a default export?

Reply:
Must do. I got this code from the tests in the project, and it works for me. I'm using it in production.

Reply:
So `inport ts from 'typescript'` does not work.

Reply:
Yeah, it does work. Intellisense suggested it to me as a refactor.